installation/README.md

```diff
@@ -156,6 +156,32 @@ it will get stuck in state "InProgress-SecretsConfig".
     ```bash
     helm install canvas oda-canvas/canvas-oda -n canvas --create-namespace 
     ```
+### 5. Optionally Install APISIX or Kong Gateway
+
+For users seeking advanced API gateway capabilities, the ODA Canvas provides the option to install either the Kong Gateway or the APISIX Gateway. Additionally, the ODA Canvas includes a specially designed Gateway Operator tailored for seamless integration and management within the ODA Canvas environment. These gateways offer powerful features for managing APIs, including traffic management, security, load balancing, rate limiting, and observability.
+
+Install APISIX Gateway and Operator
+
+APISIX is an open-source, dynamic, real-time, high-performance API gateway. The Canvas Gateway Operator streamlines the deployment and operation of APISIX within the ODA Canvas environment, ensuring smooth integration and management.
+To install the APISIX Gateway along with the APISIX Operator, run the following command:
+
+
+    helm install apisix oda-canvas/canvas-api-gateway -n ingress-apisix --create-namespace --set apisix-gateway-install.enabled=true
+
+or
+
+Install Kong Gateway and Operator
+
+Kong is a highly popular API gateway that provides a comprehensive range of features for managing APIs. The Canvas Gateway Operator simplifies the deployment and operation of the Kong Gateway within the ODA Canvas environment, ensuring efficient and reliable API management.
+
+Prerequisite: Install the Gateway API CRDs if required to enable the advanced networking capabilities required by Kong Gateway:
+    ```
+    kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
```

Comment:
There is an option to make an installation in helm a chart optional using the lookup function.

For an example see here:

https://github.com/tmforum-oda/oda-canvas/blob/master/charts/canvas-vault/templates/namespace-canvas-vault.yaml#L1-L8

If the namespace already exists, nothing is done, if it does not exist it is created but with an annotation, that it should be kept (resource-policy), otherwise with the next upgrade it would be uninstalled.

Reply:
this is required to deploy referencegrant crd if  not available in setup .thats why given as optional.

Reply:
It was just an idea, if you would like to automatically deploy the CRDs without having issues, if they already exists.
But it is also fine to keep this step manually.